server/src/main/kotlin/com/kamelia/hedera/rest/user/DiskQuotaService.kt

```diff
@@ -0,0 +1,75 @@
+package com.kamelia.hedera.rest.user
+
+import com.kamelia.hedera.database.Connection
+import com.kamelia.hedera.rest.auth.SessionManager
+import com.kamelia.hedera.util.uuid
+import com.kamelia.hedera.util.withReentrantLock
+import java.sql.Connection.TRANSACTION_REPEATABLE_READ
+import java.util.*
+import kotlinx.coroutines.sync.Mutex
+
+object DiskQuotaService {
+
+    private val quotas = mutableMapOf<UUID, Pair<Long, Long>>()
```

Comment:
create a type alias or a data class for the pair, just to make it more readable across the service which is quite complex